src/actions/appActions.js

```diff
@@ -76,6 +76,10 @@ import { getWalletFromStorage } from 'utils/wallet';
 // actions
 import { getTranslationsResourcesAndSetLanguageOnAppOpenAction } from 'actions/localisationActions';
 import { setWbtcPendingTxsAction } from './exchangeActions';
+import { REMOTE_CONFIG } from '../constants/remoteConfigConstants';
+
+//firebase
```

Comment:
```suggestion
// firebase
```

Reply:
> @divyaBacancy could you please sign the CLA please? It looks like you may have authored with three different accounts, all three accounts need to sign the agreement.

@IAmKio I have signed the CLA for divyaBacancy account. Only this account is needed will avoid working from other account(divya-svmx). admin is not my account.